Artifacts/PgSql/Structure/Admin/0130-Add-ProfileDefinition-to-Profile.sql

```diff
@@ -0,0 +1,7 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+ALTER TABLE dbo.Profiles
+    ADD ProfileDefinition VARCHAR NULL;
```

Comment:
Can you use  the XML data type to   store the XML data instead of [varchar]  like ADD [ProfileDefinition] XML NULL;

Reply:
Same as above, regarding use of XML data type.